File: articles/why-origin-stories-of-algorithms-need-care/java/OriginCareScore.java
public class OriginCareScore {
    public static void main(String[] args) {
        double[] scores = {0.96, 0.98, 0.96, 0.88, 0.98, 0.90, 0.90, 0.96, 0.98, 0.98};
        double sum = 0.0;
        for (double score : scores) sum += score;
        System.out.printf("origin_care_score=%.6f%n", sum / scores.length);
    }
}

File: articles/why-origin-stories-of-algorithms-need-care/rust/origin_care_score.rs
fn main() {
    let scores = [0.96, 0.98, 0.96, 0.88, 0.98, 0.90, 0.90, 0.96, 0.98, 0.98];
    let score: f64 = scores.iter().sum::<f64>() / scores.len() as f64;
    println!("origin_care_score={:.6}", score);
}
